### DISCUSSION TITLE
Can I store pages title and meta in database

### DISCUSSION THREAD

**Discussion by asokr** [wintercms/winter#848] [Q&A]
Hi, can you help me. how to store this data in the database, not in the theme files?

**Comment by bennothommo** [comment]
Could you explain the reason? This will allow us to perhaps help you towards a better solution.

**Comment by LukeTowers** [comment]
@asokr you might be looking for the DatabaseTemplates feature, see https://wintercms.com/docs/cms/themes#database-driven-themes
